.travis.yml

```diff
@@ -12,6 +12,7 @@ install:
 
 script:
     - vendor/bin/tester tests -s
+    - php -c tests/disable-mail.ini tests/Mail/SendmailMailer.phpt
```

Comment:
No need to add another script. You may use `@phpIni disable_function=mail` annotation in test.